### PR TITLE
Bug fix in firebase.json

### PR DESCRIPTION
**Pull request by k1ycee**
Apparently trying to deploy will fail due to a little bug just fixed it

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "hosting": {
-    "predeploy": "flutter clean; flutter build web",
+    "predeploy": "flutter clean flutter build web",
```
Comment (ditman) [inline firebase.json:3]: Thanks for sending this PR!!


I'm not sure this fixes anything? Removing the `;` results in a command that shouldn't do anything (or maybe just a flutter clean?)

Have you checked the contents of the build/web directory after this change?

**Comment by k1ycee** [comment]
Yeah so when I used the initial command it gives an error that it couldn't
run "flutter clean;" hence preventing the flutter build web from running in
the CLI and I did try it and it worked.

**Comment by k1ycee** [comment]
@googlebot I signed it!

File: firebase.json (hunk 1 above)
Comment (k1ycee) [reply firebase.json:3]: And another thing it never gets to build for the web because it gets stuck at trying to clean

Comment (ditman) [reply firebase.json:3]: @k1ycee what platform are you using to do this codelab?

**Comment by ditman** [comment]
```
dit-macbookpro:friendlyeats-flutter dit$ flutter clean flutter build web
Cleaning Xcode workspace...                                         2.3s
Deleting build...                                                    2ms
Deleting .dart_tool...                                               5ms
Deleting Generated.xcconfig...                                       0ms
Deleting flutter_export_environment.sh...                            0ms
```

The command only runs `flutter clean` and ignores everything after clean (the flutter build web part doesn't do anything). It kinds of defeats the purpose of having a clean+build step before publishing to firestore.

I can't merge this, but thanks for sending a PR! Keep them coming!  :)

**Comment by ditman** [comment]
I'm going to reopen to discuss the issue you mentioned about `flutter clean` never finishing. What OS are you using to go through the codelab?

**Comment by ditman** [comment]
This was fixed in #5 and #6, closing!

**Pull request closed without merging**